app/channels/notifications_channel.rb

```diff
@@ -0,0 +1,5 @@
+class NotificationsChannel < ApplicationCable::Channel
```

Comment:
This probably should verify that the user is authorized to perform this subscription. (Do we need to do this in the Messages one too?)